uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java

```diff
@@ -144,6 +152,7 @@ public static UiObject obtainUiObjectWithResourceIdAndText(@NonNull final String
      */
     public static UiObject obtainUiObjectWithTextAndClassType(@NonNull final String text,
                                                               @NonNull Class className) {
+        Logger.i(TAG, "Obtain an instance of the UiObject for the given text and class name..");
```

Comment:
let's also log the text and class name in the log message

Reply:
added.